include/openssl/ocsp.h

```diff
@@ -250,6 +268,8 @@ OPENSSL_EXPORT const char *OCSP_response_status_str(long status_code);
 // a string.
 OPENSSL_EXPORT const char *OCSP_cert_status_str(long status_code);
 
+// OCSP_crl_reason_str prints the OCSP CRL reason into a string.
+OPENSSL_EXPORT const char *OCSP_crl_reason_str(long s);
```

Comment:
How does this work? Where does a customer get one of these reason codes? This PR seems to be adding the 8 defines in a public header but we never use/return them.

Reply:
Ahhhh so the OCSP responder would put these values in on their side. That explains why we're not using the defines when we create the object anywhere.